src/components/dashboard/index.js

```diff
@@ -14,15 +14,34 @@ class Dashboard extends React.Component {
     }
     let userRegisteredEvents = this.props.events.filter(
       event => _.some(Object.keys(event.participants), uid => uid === this.props.user.uid));
-    return <div>
-      {this.props.user && <img className={'uk-border-circle'} src={this.props.user.photoURL} alt={'User'}/>}
-      <div>Welcome {this.props.user ? this.props.user.displayName : 'human'}.</div>
-      <div>
-        <ul>
-          {userRegisteredEvents.map(event => <li key={event.id}>
-            {event.name} <button onClick={() => this.props.unregisterFromEvent(event)}>Unregister</button></li>)}
-        </ul>
+    return <div className="dashboard">
+      <div className="user-details">
+        {this.props.user && <img className={'uk-border-circle user-img'} src={this.props.user.photoURL} alt={'User'}/>}
+        <div className="user-name">{this.props.user ? this.props.user.displayName : 'human'}.</div>
       </div>
+      <h3>Registered Events</h3>
+      <table class="uk-table uk-table-middle uk-table-divider">
+        <thead>
+            <tr>
+              <th>#</th>
+              <th>Event Name</th>
+              <th class="uk-width-small">Starts On</th>
+              <th class="uk-width-small">Registered On</th>
+              <th class="uk-width-small">Action</th>
+            </tr>
+        </thead>
+        <tbody>
+          {userRegisteredEvents.map(event =>
+            <tr key={event.id}>
+                <td>{event.id}</td>
+                <td>{event.name}</td>
+                <td>-</td>
```

Comment:
@singhpratyush whats the JSON format exactly? I need to render start time and registered on time here.